packetbeat/etc/fields.yml

```diff
@@ -836,26 +836,311 @@
+                          type: keyword
+                          description: Only present after set Global_tables_spec, the keyspace name.
+                        - name: table
+                          type: keyword
+                          description: Only present after set Global_tables_spec, the table name.
+                        - name: flags
+                          type: keyword
+                          description: Provides information on the formatting of the remaining information.
+                        - name: col_count
+                          type: integer
+                          description: Representing the number of columns selected by the query that produced this result.
+                        - name: pkey_columns
+                          type: integer
+                          description: Representing the PK columns index and counts.
+                        - name: paging_state
+                          type: keyword
+                          description: The paging_state is a bytes value that should be used in QUERY/EXECUTE to continue paging and retrieve the remainder of the result for this query.
+                - name: keyspace
+                  type: keyword
+                  description: Indicating the name of the keyspace that has been set.
+                - name: schema_change
+                  type: group
+                  description: The result to a schema_change message.
+                  fields:
+                    - name: change
+                      type: keyword
+                      description:  Representing the type of changed involved.
+                    - name: keyspace
+                      type: keyword
+                      description: This describes which keyspace has changed.
+                    - name: table
+                      type: keyword
+                      description: This describes which table has changed.
+                    - name: object
+                      type: keyword
+                      description: This describes the name of said affected object (either the table, user type, function, or aggregate name).
+                    - name: target
+                      type: keyword
+                      description: Target could be "FUNCTION" or "AGGREGATE", multiple arguments.
+                    - name: name
+                      type: keyword
+                      description: The function/aggregate name.
+                    - name: args
+                      type: keyword
+                      description: One string for each argument type (as CQL type).
+                - name: prepared
+                  type: group
+                  description: The result to a PREPARE message.
+                  fields:
+                    - name: prepared_id
+                      type: keyword
+                      description: Representing the prepared query ID.
+                    - name: req_meta
```

Comment:
why not moving `req_meta` to `cassandra.request.meta` and `prepared_id` to `cassandra.request.prepared_id`?